actors/market/src/lib.rs

```diff
@@ -810,38 +790,17 @@ impl Actor {
                         })?;
```

Comment:
As discussed, this is the point to stop continue so as to not reschedule new deals for processing, but allow deals which have already been processed once to be rescheduled.